tests/integration_tests/functional/test_logging.py

```diff
@@ -223,8 +242,5 @@ def _test_log_config(microvm, log_level="Info", show_level=True, show_origin=Tru
     microvm.start()
 
     lines = microvm.log_data.splitlines()
-    for idx, line in enumerate(lines):
-        if idx == 0:
-            assert line.startswith("Running Firecracker")
-            continue
```

Comment:
I think this message should be preserved still.

Reply:
I think how we check this depends on https://github.com/firecracker-microvm/firecracker/pull/4047#discussion_r1304524154